tensorrt_llm/quantization/quant_annotation.py

```
@@ -0,0 +1,53 @@
+    metadata for annotation
+    quant_scheme: Quantization scheme (e.g., 'int8', 'fp8')
+    backend: Quantization backend implementation
+    quant_group: Group ID for shard-aware quantization
+    metadata: Additional quantization parameters
+    """
+    quant_scheme: str
+    backend: str
+    quant_group: Optional[int] = None
+    metadata: Optional[Dict[str, Any]] = None
+
+def annotate_node(node, annotation: QuantAnnotation):
```

Ruff (0.12.2)

First line should end with a period, question mark, or exclamation point

Add closing punctuation

(D415)